test/WebJobs.Script.Tests/Managment/WebFunctionsManagerTests.cs

```diff
@@ -21,30 +21,35 @@ public class WebFunctionsManagerTests : IDisposable
         [Fact]
         public async Task VerifyDurableTaskHubNameIsAdded()
         {
-            // Setup
-            const string expectedSyncTriggersPayload = "[{\"authLevel\":\"anonymous\",\"type\":\"httpTrigger\",\"direction\":\"in\",\"name\":\"req\",\"functionName\":\"function1\"}," +
+            var vars = new Dictionary<string, string>
+            {
+                { EnvironmentSettingNames.WebSiteAuthEncryptionKey, TestHelpers.GenerateKeyHexString() },
```

Comment:
This test was setting enviroment variables but not cleaning them up, causing other tests to flake out